File: Python/Easy/Filter/Array/1913. Maximum Product Difference Between Two Pairs.py
def maxProductDifference(nums: list[int]) -> int:
    nums_sort = sorted(nums)
    return (nums_sort[-1] * nums_sort[-2]) - (nums_sort[0] * nums_sort[1])

nums = [4,2,5,9,7,4,8]
print(maxProductDifference(nums))

File: PHP/Easy/Filters/Array/1913-Maximum_Product_Difference_Between_Two_Pairs.php
<?php
// https://leetcode.com/problems/maximum-product-difference-between-two-pairs/description/
function maxProductDifference($nums)
{
    sort($nums);
    return ($nums[count($nums) - 1] * $nums[count($nums) - 2]) - ($nums[0] * $nums[1]);
}


$nums = [5, 6, 2, 7, 4];
echo maxProductDifference($nums);